.circleci/scripts/bundle-stats-commit.sh

```diff
@@ -50,11 +50,11 @@ then
     exit 0
 fi
 
-cp -R test-artifacts/chrome/mv3/bundle_size_stats.json temp/stats
+cp -R test-artifacts/chrome/mv3/bundle_size.json temp/stats
 
 echo " bundle_size_stats-${CIRCLE_SHA1}.json" >> temp/stats/fileList.txt
 
-mv temp/stats/bundle_size_stats.json "temp/stats/bundle_size_stats-${CIRCLE_SHA1}.json"
+mv temp/stats/bundle_size.json "temp/stats/bundle_size_stats-${CIRCLE_SHA1}.json"
 
 cd temp
 
```

Comment:
This issue was not caught before as these stats are generated for only develop branch